src/GameLogic/Attributes/Stats.cs

```diff
@@ -553,37 +553,37 @@ public class Stats
     public static AttributeDefinition IsStunned { get; } = new (new Guid("22C86BAF-7F27-478D-8075-E4465C2859DD"), "Is stunned", "The player is poisoned and loses health");
 
     /// <summary>
-    /// Gets the ice resistance attribute definition. Value range from 0 to 1.
+    /// Gets the ice resistance attribute definition. Value range from 0 to 0.5.
```

Comment:
Why 0.5?
I understand that there are only Rings and Pendants up to level 4 in the official game, but that could be higher for another configuration

Reply:
Yes, I didn't think of that. I'll change this and leave a "default = 0.5/max level = 4" note.